lib/mayaUsd/ufe/wrapUtils.cpp

```diff
@@ -113,13 +122,11 @@ UsdPrim ufePathToPrim(const std::string& ufePathString)
 
 void wrapUtils()
 {
-    def("getPrimFromRawItem", getPrimFromRawItem);
-
 #ifdef UFE_V2_FEATURES_AVAILABLE
+    def("getPrimFromRawItem", getPrimFromRawItem);
     def("getNodeNameFromRawItem", getNodeNameFromRawItem);
-#endif
-
     def("getNodeTypeFromRawItem", getNodeTypeFromRawItem);
+#endif
```

Comment:
The raw item (for ufe scene item) was only added in UFE v2. So these three wrappers that use raw item should only be avail in v2.